src/Components/Steps/HouseholdSize.tsx

```diff
@@ -0,0 +1,111 @@
+import { TextField } from '@mui/material';
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import { Context } from '../Wrapper/Wrapper';
+import { useContext } from 'react';
+import * as z from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import ErrorMessageWrapper from '../ErrorMessage/ErrorMessageWrapper';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { getStepNumber } from '../../Assets/stepDirectory';
+import QuestionHeader from '../QuestionComponents/QuestionHeader';
+import QuestionQuestion from '../QuestionComponents/QuestionQuestion';
+import PrevAndContinueButtons from '../PrevAndContinueButtons/PrevAndContinueButtons';
+import { useNavigate, useParams } from 'react-router-dom';
+import { updateScreen } from '../../Assets/updateScreen';
+import { useGoToNextStep } from '../QuestionComponents/questionHooks';
+import HelpButton from '../HelpBubbleIcon/HelpButton';
+
+const HouseholdSize = () => {
+  const { formData, setFormData, locale } = useContext(Context);
+  const { uuid } = useParams();
+  const navigate = useNavigate();
+  const currentStepId = getStepNumber('householdSize', formData.immutableReferrer);
+  const backNavigationFunction = () => navigate(`/${uuid}/step-${currentStepId - 1}`);
+  const nextStep = useGoToNextStep('householdSize', '1');
+  const intl = useIntl();
+
+  const requiredErrorProps = {
+    id: 'hhSize.requiredError',
+    defaultMsg: 'Household size is required',
+  };
+  const invalidTypeErrorProps = {
+    id: 'hhSize.invalidTypeError',
+    defaultMsg: 'Household size must be a number',
+  };
+
+  const formSchema = z.object({
+    householdSize: z.coerce
+      .number({
+        required_error: intl.formatMessage(requiredErrorProps),
+        invalid_type_error: intl.formatMessage(invalidTypeErrorProps),
+      })
+      .positive()
```

Comment:
Can you enter 0?

Reply:
Yes, but zod will render an error message and not allow the user to continue.